habitat_baselines/rl/ppo/ppo.py

```diff
@@ -4,28 +4,37 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+from typing import Optional, Tuple, Union
+
 import torch
+from torch import Tensor
 from torch import nn as nn
 from torch import optim as optim
 
+from habitat_baselines.common.rollout_storage import RolloutStorage
+from habitat_baselines.rl.ddppo.policy.resnet_policy import (
+    PointNavResNetPolicy,
+)
+from habitat_baselines.rl.ppo.policy import PointNavBaselinePolicy
+
 EPS_PPO = 1e-5
 
 
 class PPO(nn.Module):
     def __init__(
         self,
-        actor_critic,
-        clip_param,
-        ppo_epoch,
-        num_mini_batch,
-        value_loss_coef,
-        entropy_coef,
-        lr=None,
-        eps=None,
-        max_grad_norm=None,
-        use_clipped_value_loss=True,
-        use_normalized_advantage=True,
-    ):
+        actor_critic: Union[PointNavResNetPolicy, PointNavBaselinePolicy],
```

Comment:
```suggestion
        actor_critic: Policy,
```

The base class should work here